engines/producers/db/migrate/20150815082649_create_products.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProducts < ActiveRecord::Migration
+  def change
+    create_table :products do |t|
+      t.string :name, null: false
+      t.integer :price, null: false
+      t.string :unit, default: 0, null: false
```

Comment:
BTW, this should be integer, right? Not stringa :trollface: 

http://api.rubyonrails.org/v4.1.0/classes/ActiveRecord/Enum.html